2022-02-18T17:59:30.364Z error: Unexpected token s in JSON at position 68 Unexpected token s in JSON at position 68
2022-02-18T18:04:49.239Z error: Unexpected token s in JSON at position 68 Unexpected token s in JSON at position 68
2022-02-18T18:05:18.228Z error: Unexpected token 
 in JSON at position 65 Unexpected token 
 in JSON at position 65
2022-02-18T18:09:38.922Z error: Unexpected string in JSON at position 73 Unexpected string in JSON at position 73
2022-02-18T18:10:27.560Z error: Unexpected string in JSON at position 73 Unexpected string in JSON at position 73
2022-02-19T00:41:10.946Z error: uncaughtException: Configuration property "database_naame" is not defined
Error: Configuration property "database_naame" is not defined
    at Config.get (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/node_modules/config/lib/config.js:182:11)
    at Object.<anonymous> (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/services/dbService.js:10:20)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/index.js:7:1)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
2022-02-19T00:41:10.946Z error: uncaughtException: Configuration property "database_naame" is not defined
Error: Configuration property "database_naame" is not defined
    at Config.get (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/node_modules/config/lib/config.js:182:11)
    at Object.<anonymous> (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/services/dbService.js:10:20)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/index.js:7:1)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
2022-02-19T00:45:37.638Z error: Unexpected string in JSON at position 73 Unexpected string in JSON at position 73
2022-02-19T00:49:36.448Z error: Cannot read properties of null (reading 'role') Cannot read properties of null (reading 'role')
2022-02-26T15:54:59.349Z error: console is not a function console is not a function
2022-02-26T15:55:07.829Z error: console is not a function console is not a function
2022-02-26T16:56:10.217Z error: Assignment to constant variable. Assignment to constant variable.
2022-02-26T17:06:23.357Z error: sql is not defined sql is not defined
2022-02-26T17:20:29.239Z error: uncaughtException: Malformed arrow function parameter list
/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/routes/taxonomy.js:100
router.post("/insert_synonym", [auth,admin] (req, res) => {
                                            ^^^^^^^^^

SyntaxError: Malformed arrow function parameter list
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1031:15)
    at Module._compile (node:internal/modules/cjs/loader:1065:27)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/services/routes.js:2:18)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
2022-02-26T17:20:29.239Z error: uncaughtException: Malformed arrow function parameter list
/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/routes/taxonomy.js:100
router.post("/insert_synonym", [auth,admin] (req, res) => {
                                            ^^^^^^^^^

SyntaxError: Malformed arrow function parameter list
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1031:15)
    at Module._compile (node:internal/modules/cjs/loader:1065:27)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/services/routes.js:2:18)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
2022-02-26T17:51:15.552Z error: uncaughtException: Cannot set headers after they are sent to the client
Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:371:5)
    at ServerResponse.setHeader (node:_http_outgoing:576:11)
    at ServerResponse.header (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/node_modules/express/lib/response.js:776:10)
    at ServerResponse.send (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/node_modules/express/lib/response.js:170:12)
    at ServerResponse.json (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/node_modules/express/lib/response.js:267:15)
    at ServerResponse.send (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/node_modules/express/lib/response.js:158:21)
    at Query.onResult (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/routes/taxonomy.js:242:16)
    at /Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/node_modules/mysql2/lib/commands/query.js:90:16
    at processTicksAndRejections (node:internal/process/task_queues:78:11)
2022-02-26T17:51:15.552Z error: uncaughtException: Cannot set headers after they are sent to the client
Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:371:5)
    at ServerResponse.setHeader (node:_http_outgoing:576:11)
    at ServerResponse.header (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/node_modules/express/lib/response.js:776:10)
    at ServerResponse.send (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/node_modules/express/lib/response.js:170:12)
    at ServerResponse.json (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/node_modules/express/lib/response.js:267:15)
    at ServerResponse.send (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/node_modules/express/lib/response.js:158:21)
    at Query.onResult (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/routes/taxonomy.js:242:16)
    at /Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/node_modules/mysql2/lib/commands/query.js:90:16
    at processTicksAndRejections (node:internal/process/task_queues:78:11)
2022-02-26T18:14:54.541Z error: uncaughtException: ERROR: Access denied for user 'root'@'localhost' (using password: YES)Error: Access denied for user 'root'@'localhost' (using password: YES)
Error: ERROR: Access denied for user 'root'@'localhost' (using password: YES)Error: Access denied for user 'root'@'localhost' (using password: YES)
    at C:\taxonomy\official\Backend\services\dbService.js:15:11
    at C:\taxonomy\official\Backend\node_modules\mysql2\lib\pool.js:60:18
    at PoolConnection.<anonymous> (C:\taxonomy\official\Backend\node_modules\mysql2\lib\connection.js:775:13)
    at Object.onceWrapper (node:events:510:26)
    at PoolConnection.emit (node:events:402:35)
    at PoolConnection._notifyError (C:\taxonomy\official\Backend\node_modules\mysql2\lib\connection.js:236:12)
    at ClientHandshake.<anonymous> (C:\taxonomy\official\Backend\node_modules\mysql2\lib\connection.js:125:14)
    at ClientHandshake.emit (node:events:402:35)
    at ClientHandshake.execute (C:\taxonomy\official\Backend\node_modules\mysql2\lib\commands\command.js:39:14)
    at PoolConnection.handlePacket (C:\taxonomy\official\Backend\node_modules\mysql2\lib\connection.js:456:32)
2022-02-26T18:14:54.541Z error: uncaughtException: ERROR: Access denied for user 'root'@'localhost' (using password: YES)Error: Access denied for user 'root'@'localhost' (using password: YES)
Error: ERROR: Access denied for user 'root'@'localhost' (using password: YES)Error: Access denied for user 'root'@'localhost' (using password: YES)
    at C:\taxonomy\official\Backend\services\dbService.js:15:11
    at C:\taxonomy\official\Backend\node_modules\mysql2\lib\pool.js:60:18
    at PoolConnection.<anonymous> (C:\taxonomy\official\Backend\node_modules\mysql2\lib\connection.js:775:13)
    at Object.onceWrapper (node:events:510:26)
    at PoolConnection.emit (node:events:402:35)
    at PoolConnection._notifyError (C:\taxonomy\official\Backend\node_modules\mysql2\lib\connection.js:236:12)
    at ClientHandshake.<anonymous> (C:\taxonomy\official\Backend\node_modules\mysql2\lib\connection.js:125:14)
    at ClientHandshake.emit (node:events:402:35)
    at ClientHandshake.execute (C:\taxonomy\official\Backend\node_modules\mysql2\lib\commands\command.js:39:14)
    at PoolConnection.handlePacket (C:\taxonomy\official\Backend\node_modules\mysql2\lib\connection.js:456:32)
2022-02-28T16:00:48.915Z error: uncaughtException: ERROR: Access denied for user 'root'@'localhost' (using password: YES)Error: Access denied for user 'root'@'localhost' (using password: YES)
Error: ERROR: Access denied for user 'root'@'localhost' (using password: YES)Error: Access denied for user 'root'@'localhost' (using password: YES)
    at /Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/services/dbService.js:15:11
    at /Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/node_modules/mysql2/lib/pool.js:60:18
    at PoolConnection.<anonymous> (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/node_modules/mysql2/lib/connection.js:775:13)
    at Object.onceWrapper (node:events:510:26)
    at PoolConnection.emit (node:events:402:35)
    at PoolConnection._notifyError (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/node_modules/mysql2/lib/connection.js:236:12)
    at ClientHandshake.<anonymous> (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/node_modules/mysql2/lib/connection.js:125:14)
    at ClientHandshake.emit (node:events:402:35)
    at ClientHandshake.execute (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/node_modules/mysql2/lib/commands/command.js:39:14)
    at PoolConnection.handlePacket (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/node_modules/mysql2/lib/connection.js:456:32)
2022-02-28T16:00:48.915Z error: uncaughtException: ERROR: Access denied for user 'root'@'localhost' (using password: YES)Error: Access denied for user 'root'@'localhost' (using password: YES)
Error: ERROR: Access denied for user 'root'@'localhost' (using password: YES)Error: Access denied for user 'root'@'localhost' (using password: YES)
    at /Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/services/dbService.js:15:11
    at /Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/node_modules/mysql2/lib/pool.js:60:18
    at PoolConnection.<anonymous> (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/node_modules/mysql2/lib/connection.js:775:13)
    at Object.onceWrapper (node:events:510:26)
    at PoolConnection.emit (node:events:402:35)
    at PoolConnection._notifyError (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/node_modules/mysql2/lib/connection.js:236:12)
    at ClientHandshake.<anonymous> (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/node_modules/mysql2/lib/connection.js:125:14)
    at ClientHandshake.emit (node:events:402:35)
    at ClientHandshake.execute (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/node_modules/mysql2/lib/commands/command.js:39:14)
    at PoolConnection.handlePacket (/Users/apostoloskapetanios/Library/Mobile Documents/com~apple~CloudDocs/Knowledge/Programming/Tzakos/Tzakos_Taxonomy/Backend/node_modules/mysql2/lib/connection.js:456:32)
2022-02-28T20:14:08.041Z error: Unexpected token d in JSON at position 0 Unexpected token d in JSON at position 0
2022-02-28T20:15:52.930Z error: Unexpected token d in JSON at position 0 Unexpected token d in JSON at position 0
2022-02-28T20:15:56.310Z error: Unexpected token d in JSON at position 0 Unexpected token d in JSON at position 0
2022-02-28T20:16:20.250Z error: Unexpected token d in JSON at position 0 Unexpected token d in JSON at position 0
